src/Scaffolding/VS.Web.CG.Mvc/Templates/Identity/Bootstrap4/wwwroot/css/site.css

```diff
@@ -42,24 +41,27 @@ button.accept-policy {
   line-height: inherit;
 }
 
-/* Sticky footer styles
--------------------------------------------------- */
-html {
-  position: relative;
-  min-height: 100%;
+.header {
+  background-color: var(--bs-body-bg);
 }
 
-body {
-  /* Margin bottom by footer height */
-  margin-bottom: 60px;
-}
 .footer {
-  position: absolute;
-  bottom: 0;
-  width: 100%;
-  overflow: scroll;
+  background-color: var(--bs-body-bg);
   white-space: nowrap;
-  /* Set the fixed height of the footer here */
-  height: 60px;
-  line-height: 60px; /* Vertically center the text there */
+  line-height: 45px;
+}
+
+@media (min-width: 576px) {
+
+  body {
+    margin-bottom: 45px;
+  }
+
+  .footer {
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+    z-index: 9;
```

Comment:
Why is z-index necessary? Because the footer is now fixed? Do we really want a fixed footer?